packages/hoprd/src/api/v2/paths/sign.ts

```diff
@@ -0,0 +1,44 @@
+import { Operation } from 'express-openapi'
+import { u8aToHex } from '@hoprnet/hopr-utils'
+
+export const parameters = []
+
+export const POST: Operation = [
+  async (req, res, _next) => {
+    try {
+      const signedMessage = await req.context.node.signMessage(req.body.body)
```

Comment:
Its confusing that I give `body` as input and receive `signedMessage` as output. In that case the input should be `message`.

Reply:
Ok, that makes sense, updating.

Reply:
![image](https://user-images.githubusercontent.com/1128312/148941301-bb038a48-0521-45f5-9d55-acd318c4fb7f.png)

Seems like I got this from `/messages`, which we parse as `message` from `body`

![image](https://user-images.githubusercontent.com/1128312/148941412-3a3b8a02-6d40-409f-bd12-07d8c2e6c725.png)

Probably that should be then be `req.body.message` as well, but we can change it in a different PR.

Reply:
Updated in 23a520e